### PR TITLE
Restructure Damon/Agent messages

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

This PR greatly simplifies how we send messages over Bastion. Instead of hitting a different signalr endpoint for each hub, now we just send all the data to the same hub and let a background poller process the logs (i.e. deserialize and determine if they need to be logged). 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactor (code change with no functionality change)
- [ ] Enhancement (minor change below the level of a feature)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Remove feature (removes a feature we don't support anymore)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related JIRA tickets

Relates to JIRA: CWC-1129

## Relevant release note information

Release Notes:

## Potential Security Impacts

Does this PR have any security impact?
Does it fix a security issue?
Does it add attack surface?
Why do we think this change is safe?

### Development

- [ ] Does the code changed/added as part of this pull request have test coverage?
- [ ] Do all tests related to the changed code pass in development?
- [ ] Have you ensured that at least one other person has tested your code?
- [ ] Have you tested the code?
- [ ] Have you attached any pictures (if relevant)?

### CODE REVIEW

File: kube-go-assets/bctl/daemon/daemon.go [hunk 1]
```diff
@@ -67,18 +67,13 @@ func targetSelectHandler(agentMessage wsmsg.AgentMessage) (string, error) {
 	if err := json.Unmarshal(agentMessage.MessagePayload, &payload); err == nil {
 		if p, ok := payload["keysplittingPayload"].(map[string]interface{}); ok {
 			switch p["action"] {
-			case "kube/restapi":
-				return "RequestDaemonToBastion", nil
+			case "kube/restapi/request":
```
Comment (sidpremkumar) [inline kube-go-assets/bctl/daemon/daemon.go:70]: Not sure this case sytax will work, we should just make them inline
```
case "1", "2": 
```

File: kube-go-assets/bctl/agent/plugin/kube/actions/restapi/restapi.go [hunk 2]
```diff
@@ -10,6 +10,11 @@ import (
 	lggr "bastionzero.com/bctl/v1/bzerolib/logger"
 )
 
+const (
+	RestResponse = "kube/restapi/response"
+	RestRequest  = "kube/restapi/request"
```
Comment (lgmugnier) [inline kube-go-assets/bctl/agent/plugin/kube/actions/restapi/restapi.go:15]: I don't think we need this distinction.  Daemon should know it's a response and Agent knows it's a request, right?

Comment (sidpremkumar) [reply kube-go-assets/bctl/agent/plugin/kube/actions/restapi/restapi.go:15]: We need this distinction bc we need to differentiate in Webshell backend when we parse the logs if it's a request or a response.

Furthermore, it makes debugging really hard if the same message type is being used in two different places.

**Comment by sidpremkumar** [comment]
This should be a major agent/daemon version bump

**Review by lgmugnier — APPROVED** [review]

**Pull request merged**